recipes/s3transfer/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "s3transfer" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "af2e541ac584a1e88d3bca9529ae784d2b25e5d448685e0ee64f4c0e1e017ed2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+#    - botocore >=1.3.0,<2.0.0
+
+  run:
+    - python
+    - botocore >=1.3.0,<2.0.0
```

Comment:
Relies on [`futures>=2.2.0,<4.0.0`]( https://github.com/boto/s3transfer/blob/0.1.3/setup.py#L42 ) for Python 2.x. Please add with a `py2k` selector.